src/Infrastructure/Services/ContentfulService.cs

```diff
@@ -0,0 +1,24 @@
+using System.Threading.Tasks;
+using Application.Contracts;
+using Application.Responses.V1;
+using Contentful.Core;
+using Contentful.Core.Search;
+
+namespace Infrastructure.Services
+{
+    public class ContentfulService: IContentfulService
+    {
+        private readonly IContentfulClient _client;
+
+        public ContentfulService(IContentfulClient client)
+        {
+            _client = client;
+        }
+
+        public async Task<ContentfulEmail> GetContentfulEmailAsync(string entryId, string locale = "en-GB")
+        {
+            var entry = await _client.GetEntry(entryId, new QueryBuilder<ContentfulEmail>().LocaleIs(locale));
```

Comment:
`LocaleIs()` should accept `CultureInfo locale` directly without having to call `ToString()`.

Reply:
Great thanks, this was my only question around using CultureInfo

Reply:
I had to ToString, it wouldn't accept CultureInfo